Resources/doc/filters.rst

```diff
@@ -124,39 +124,67 @@ You can now reference and use your custom filter when defining filter sets in yo
 Dynamic filters
 ---------------
 
-With a custom controller action it is possible to dynamically modify the
-configuration that will be applied to the image. Inside the controller you can
-access ``FilterManager`` instance, pass configuration as third parameter of
-``applyFilter`` method (for example based on information associated with the
-image or whatever other logic you might want).
+It is possible to dynamically modify the configuration that will be applied
+to the image, by passing configuration as third parameter to ``applyFilter``.
 
-A simple example showing how to change the filter configuration dynamically.
+First you need to set some aliases in your ``services.yaml`` so enable
+autowiring:
+
+.. code-block:: yaml
+
+    # config/services.yaml
+    
+    services:
+        Liip\ImagineBundle\Imagine\Cache\CacheManager:
+            alias: liip_imagine.cache.manager
+        Liip\ImagineBundle\Imagine\Data\DataManager:
+            alias: liip_imagine.data.manager
+        Liip\ImagineBundle\Imagine\Filter\FilterManager:
```

Comment:
these alias are added in #1165 - i'd remove this part and only keep the service.

Reply:
OK, great, I've updated it :-)